src/Illuminate/Foundation/Console/DownCommand.php

```diff
@@ -64,13 +69,40 @@ public function handle()
     protected function getDownFilePayload()
     {
         return [
-            'time' => $this->currentTime(),
-            'message' => $this->option('message'),
+            'redirect' => $this->redirectPath(),
             'retry' => $this->getRetryTime(),
-            'allowed' => $this->option('allow'),
+            'secret' => $this->option('secret'),
+            'status' => (int) $this->option('status', 503),
+            'template' => $this->option('render') ? $this->prerenderView() : null,
         ];
     }
 
+    /**
+     * Get the path that users should be redirected to.
+     *
+     * @return string
+     */
+    protected function redirectPath()
+    {
+        if ($this->option('redirect') && $this->option('redirect') !== '/') {
```

Comment:
Note - looks good but generally not a fan of directly comparing strings, would prefer str cmp methods instead

Reply:
Why?

FTR, there's exactly _one_ `strcmp` in the entire framework AFAICS. I didn't bother to count the direct string comparison checks 😄